samples: bluetooth: mesh: dfu: Add DFU srv shell cmds for cancelling dfu (#11612)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/dfu/target/README.rst

For more information about the firmware distribution process, see :ref:`zephyr:bluetooth_mesh_dfu`.

Interacting with the sample through shell
-----------------------------------------

> This sample enables the Firmware Update Server model shell commands using the :kconfig:`CONFIG_BT_MESH_SHELL_DFU_SRV` option.

Reviewer: ```suggestion
This sample enables the Firmware Update Server model shell commands using the :kconfig:option:`CONFIG_BT_MESH_SHELL_DFU_SRV` Kconfig option.
```

> This allows to cancel the ongoing DFU transfer using `mesh models dfu srv rx-cancel` shell command if it gets stalled.

Reviewer: ```suggestion
This allows cancelling of an ongoing DFU transfer using the ``mesh models dfu srv rx-cancel`` shell command, if the ongoing DFU transfer gets stalled.

```

> This allows canelling of an ongoing DFU transfer using the ``mesh models dfu srv rx-cancel`` shell command, if the ongoing DFU transfer gets stalled.

Reviewer 2: ```suggestion
This allows cancelling of an ongoing DFU transfer using the ``mesh models dfu srv rx-cancel`` shell command, if the ongoing DFU transfer gets stalled.
```
